src/webgl/p5.Geometry.js

```diff
@@ -185,17 +185,142 @@ p5.Geometry = class Geometry {
     return n.mult(Math.asin(sinAlpha) / ln);
   }
   /**
- * computes smooth normals per vertex as an average of each
- * face.
- * @method computeNormals
- * @chainable
- */
-  computeNormals() {
+   * This function calculates normals for each face, where each vertex's normal is the average of the normals of all faces it's connected to.
+   * i.e computes smooth normals per vertex as an average of each face.<br>
```

Comment:
Generally in other docs, rather than explicitly adding a `<br>`, we just add a blank line between paragraphs (well, blank except for the `*` from the doc comment.) Maybe we can do this here too?